### ISSUE TITLE
[Feature Request] Add option to disable PR comment

### ISSUE THREAD

**Issue by oscard0m** [calibreapp/image-actions#172]
### What problem would this feature solve?

There is no way to disable PR comments (https://github.com/calibreapp/image-actions/blob/20f376152ba330053aaf2f53342654551348df16/src/index.ts#L36) but keep the logic of the commit and report. This causes some noise to reviewers on a draft PR with changes in progress.

### Describe the solution you’d like to see

I would like to add an option to disable PR comments (https://github.com/calibreapp/image-actions/blob/20f376152ba330053aaf2f53342654551348df16/src/index.ts#L36) but keep the logic of the commit and report.

### Are you willing to work on implementing this solution?

Yes

### Describe alternatives you’ve considered

Considering there is a `config` already in place and other existing options with similar logic (like `compressOnly`) I did not consider other options.

### Code of Conduct

- [X] I agree to follow this project’s Code of Conduct

**Comment by benschwarz** [comment]
As I understand the report @oscard0m, you're wanting to use image-actions to compress images & commit them, but not post any details to the PR? 

For this workflow, you may want to [check the readme which has a recipe](https://github.com/calibreapp/image-actions#process-pull-requests-from-forked-repositories) using `compressOnly`. You could potentially use this as a starting point to update the action. 

Coming from another angle — maybe it would be better to NOT run image-actions at all, until the PR has been moved from `draft` to `open`? You can detect draft PRs using something like: `if: github.event.pull_request.draft == false`.

**Comment by oscard0m** [comment]
> As I understand the report @oscard0m, you're wanting to use image-actions to compress images & commit them, but not post any details to the PR? Is that correct?

Correct.

> For this workflow, you may want to [check the readme which has a recipe](https://github.com/calibreapp/image-actions#process-pull-requests-from-forked-repositories) using `compressOnly`. You could potentially use this as a starting point to update the action.

Thanks for pointing this out. I like the approach of running the workflow on `push` to `main` branch and opening its corresponding PR, but this opens the door to land images which are not compressed, which is something it would be nice to avoid inside the same PR.

> Coming from another angle — maybe it would be better to NOT run image-actions at all, until the PR has been moved from `draft` to `open`? You can detect draft PRs using something like: `if: github.event.pull_request.draft == false`.

This is a great idea! I will set this up in the meantime.

---

@benschwarz
Do you think adding a `config.skipComment` option would be an interesting feature or it will bring more problems/confusion than benefits to users? What are your thoughts?

**Comment by benschwarz** [comment]
@oscard0m I'm happy with how it works at the moment. Folks may want to customise how image-actions interacts with their repos, which is totally understandable & 100% valid. At the same time, if it doesn't work for them, they're also welcome to fork it, build their own system using image-actions code, or find a another alternative. Offering configuration options to opt out of core functionality seems like a good place to draw the line. 

tldr, I probably wouldn't? 🤷‍♂️

**Comment by benschwarz** [comment]
@oscard0m I guess my recommendation is — see if `compressOnly` (and image-actions in general) works out for you, and if it doesn't, please post back here 😊

**Comment by oscard0m** [comment]
Thanks, @benschwarz.

I will use the approach suggested in the documentation of running the action in the `main` branch with `compressOnly` and opening a PR if images can be compressed.

I will probably experiment with a fork, and if I come up with valuable changes that could make sense in this repo, I will open a PR for it.

Thanks for your assistance on this. I am closing this issue until further notice.

**Comment by benschwarz** [comment]
Thanks @oscard0m!
